src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,21 @@
+.catalog {
+  box-sizing: content-box;
```

Comment:
can we rewrite it to `border-box` to use one `box-sizing` in the project

Reply:
So if we already used border-box with * selector, it's beter not to use other box-sizing values? 
@dffUqp